handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+
+!!! note "Managed NI LinuxRT Nodes"
+    NI recommendations changing the default username and password on the target. These credentials are required to SSH into the target. These credentials are required when a SystemLink server adds a Linux RT target to its collection of managed systems.
+
+When a target is approved by SystemLink and becomes a managed node, SystemLink securely transfers configuration, certificates, and credentials needed to authenticate with the SystemLink server's [role-based access control system](/rbac/rbac/). SystemLink client APIs include an [**Auto** configuration](https://www.ni.com/documentation/en/systemlink/latest/systemlink-labview-node-ref/open-configuration-http-auto/) automatically consumes these credentials. This prevents the need to include credentials and other secrets in your test application code.
+
+!!! warn "Do not expose Salt ports to the public internet"
+
+    Due to the capabilities of Salt, users are encourage to configure firewalls and appropriate CIDR blocks to prevent exposing Salt ports (4505 and 4506) to the public internet.
+
+## TLS for remote MongoDB Instances
+
+MongoDB support TLS connections. Refer to the [MongoDB manual](https://docs.mongodb.com/manual/tutorial/configure-ssl/) for details on enabling TLS. NI recommends enabling TLS for remote MongoDB connections.
+
+!!! note
+
+    SystemLink does not support `mongod` instances configured with [client certificate validation](https://docs.mongodb.com/manual/tutorial/configure-x509-client-authentication/). Ensure your `mongod` is started without this requirement to allow SystemLink to successfully connect.
+
+### Connecting to MongoDB Using Self-signed certificates
+
+!!! warn MongoDB with self-signed certificates
+    Self signed certificates should be used for testing purposes only.
+
+Refer, to the [MongoDB manual](https://docs.mongodb.com/manual/appendix/security/appendixA-openssl-ca/) for steps to create and run `mongod` with a self-signed certificate.
+
+!!! note "Running MongoDB without Client certificate validation"
+
+    Start `mongod` with command line arguments:
+    ```bash
+    mongod --tlsMode requireTLS --tlscertificateKeyFile /etc/ssl/test-server1.pem --port 27017 --dbpath /var/lib/mongo --bind_ip_all
+    ```
+
+    Start `mongod` with a Mongo Configuration file:
+    ```yaml
+    net:
+        port: 27017
+        bindIp: 0.0.0.0
+        tls:
+            mode: requireTLS
+            certificateKeyFile: /etc/ssl/test-server1.pem
+    systemLog:
+        destination: file
+        logAppend: true
+        path: /var/log/mongodb/mongod.log
+    storage:
+        dbPath: /var/lib/mongo
+        journal:
+            enabled: true
+    processManagement:
+        fork: true  #fork and run in background
+        pidFilePath: /var/run/mongodb/mongod.pid  # location of pidfile
+        timeZoneInfo: /usr/share/zoneinfo
+    ```
+
+#### Installing MongoDB Self-Signed certificates
```

Comment:
Is this all the documentation we'll need for this feature? https://ni.visualstudio.com/DevCentral/_workitems/edit/1094237

Reply:
No, I think there is more we will want to document; especially with regards to connecting to managed instances such at Atlas.